transforms/universal/filter/ray/Makefile

```diff
@@ -46,13 +46,9 @@ run-cli-sample:
 		--filter_columns_to_drop \"[ 'extra', 'cluster' ]\" "	\
                 .transforms.run-src-file
 
+run-local-sample: .transforms.run-local-ray-sample
 
-
-run-local-python-only-sample: .transforms.run-local-sample
-
-run-local-sample: .transforms.run-local-sample
-
-run-s3-sample: .transforms.run-s3-sample
+run-s3-sample: .transforms.run-s3-ray-sample
 
 minio-start:	.minio-start
 
```

Comment:
Can we fix minio-start as well. Currently it does not copy additional files beyond input

Reply:
can we make this a separate PR.  its pretty orthogonal to these changes

Reply:
We can, but, we talked about it several times and its still not there

Reply:
Submit an issue and don't poke arbitrary fingers at random PRs over this.  Make an issue and fix it!

Reply:
Well, `minio-start` is currently broken. In addition to what I noted here before its also always getting data from ray directory. So I am not convinced we should merge a PR relying on the broken things

Reply:
The issue https://github.com/IBM/data-prep-kit/issues/65 is covering this problem, but it has been marked as fixed.  Do we need to re-open that issue again?